tests/test_acasclient.py

```diff
@@ -2728,14 +2766,19 @@ def get_basic_experiment_load_file_with_project(project_code, tempdir, corp_name
         data_file_contents = f.read()
 
     # Replace the project code
-    data_file_contents = data_file_contents.replace('Global', project_code)
+    if project_code is not None:
+        data_file_contents = data_file_contents.replace('Global', project_code)
 
     # If corp name is specified, replace the corp name
     if corp_name is not None:
         data_file_contents = data_file_contents.replace('CMPD-0000001-001', corp_name)
 
+    # If scientist is specified, replace the scientist
+    if scientist is not None:
+        data_file_contents = data_file_contents.replace('bob', scientist)
+
     # Write the data file to the temp dir
-    file_name = f'basic-experiment-with-project-{project_code}.csv'
+    file_name = f'basic-experiment-{ str(uuid.uuid4())}.csv'
```

Comment:
Wasn't a requirement that this function returned a project code in the name of the output file so I just added uuid to make sure it got a unique name.